DynmapCore/src/main/resources/extracted/web/js/map.js

```diff
@@ -278,12 +278,17 @@ DynMap.prototype = {
 					worldsadded[wname] = true;
 				}
 
+				var worldName = wname;
+				if(wname.startsWith('world_')) {
```

Comment:
This does not work with non-default world names (world is just the default)

Reply:
No problem.

Will take another look at it, soon.